src/lib/query/iface.rs

```diff
@@ -568,3 +573,38 @@ pub(crate) fn fill_bridge_vlan_info(
     }
     Ok(())
 }
+
+// Currently there is no valid netlink way to get the driver information as the
+// ETHTOOL_GDRVINFO ioctl command has no netlink equivalent. We use sysfs content
+// /sys/class/net/<if_name>/device/ and extract the last element from the "driver"-link
+// (https://docs.kernel.org/admin-guide/sysfs-rules.html)
+fn _get_iface_driver(if_name: &str) -> Option<String> {
```

Comment:
In rust, there is no need to use `_` as indicator of private function because rust has `pub` and `pub(crate)`, every function without them are private to current file.

Reply:
Fair, I did it this way because just above there was already existing `fn _get_iface_name(nl_msg: &LinkMessage) -> String` function